### PR TITLE
remove all references to Bruce's email address

### PR DESCRIPTION
**Pull request by bruceravel**
I'd like to encourage gnuplot.el users to open issues here at Github rather than sending email directly to me.  That is particularly true given that I do so little work on this code these days.

So, this PR removes all instances of my email address, replacing it where appropriate with the URL for this repository and a suggestion to use the issues.

This PR also adds Jon to the author list in gnuplot.el, which seems right since he has been doing the lion's share of the work for a really long time.  In fact, I would be open to transferring ownership to Jon, but that's not an issue for this PR....

### CODE REVIEW

**Comment by tarsius** [comment]
I recommend you begin with `git reset HEAD~2`. Then selectively stage to create three new commits. The last two of these commits should be attributed to you. The first probably to Bruce.

1. Jon as the author.

2. **Remove** `gnuplot-bug-report` and all uses of it. Do not just comment it out--it is of no use now and if we ever want it back it is easy to locate a commit whose summary is `Remove gnuplot-but-report`.

3. Replace the email address with the new url.

File: gnuplot.el [hunk 1]
```diff
@@ -3206,7 +3205,7 @@ maintainer of `gnuplot-mode'."
 This was written with version 4.6 of gnuplot in mind, but should
 work with newer and older versions.
 
-Report bugs in `gnuplot-mode' using \\[gnuplot-bug-report].
+Report bugs in `gnuplot-mode' as issues at https://github.com/emacsorphanage/gnuplot
```
Comment (tarsius) [inline gnuplot.el]: How about 

```
Report bugs in `gnuplot-mode' at https://github.com/emacsorphanage/gnuplot/issues
```

or even just

```
Report bugs at https://github.com/emacsorphanage/gnuplot/issues
```

File: gnuplot.el [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 ;; Copyright (C) 1998, 2011 Phil Type and Bruce Ravel, 1999-2012 Bruce Ravel
 
-;; Author:     Bruce Ravel <bruceravel1@gmail.com> and Phil Type
-;; Maintainer: Bruce Ravel <bruceravel1@gmail.com>
+;; Authors:    Jon Oddie, Bruce Ravel, Phil Type, and others
```
Comment (tarsius) [inline gnuplot.el]: The function `lm-authors` (from the built-in `lisp-mnt.el` library) has to be able to extract this, and it is not a very good function. (I'll should finally get around to contributing a better implementation.) As a result you have to spell it like this (multiple lines, singular "Author"):

```emacs-lisp
;; Author: Jon Oddie
;;   Bruce Ravel
;;   Phil Type
;;   and others
```

I would omit the `and others`.

**Comment by conao3** [comment]
Create another PR #41.

File: gnuplot.el (hunk 1 above)
Comment (tarsius) [reply gnuplot.el]: Did you see this?

Comment (conao3) [reply gnuplot.el]: Yes.
But I recheck my commit, I've forgotten to apply your suggestions in some places.

**Comment by conao3** [comment]
Close because another PR merged (#41).

**Pull request closed without merging**